account_invoice_merge_payment/README.rst

```diff
@@ -0,0 +1,38 @@
+Account Invoice Merge Payment
+=============================
+
+This module was written to extend the functionality of Account Invoice Merge
+module to support fields added in Account Payment Partner from OCA/bank-payment
+
+Installation
+============
+
+To install this module, you need to:
+
+ * Click on install button
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
```

Comment:
I'd update this section to simply say the module auto installs when account_invoice_merge and account_payment_partner are installed.